cvat-core/src/server-proxy.js

```diff
@@ -465,6 +465,63 @@
                 });
             }
 
+            async function backupTask(id) {
```

Comment:
Why use "backup" name anywhere in the PR? Opposite of "import" is "export" (as in PR title)